.eslintrc.json

```diff
@@ -0,0 +1,108 @@
+{
+  "root": true,
+  "ignorePatterns": [
+    "projects/**/*",
```

Comment:
Does this path exist?

Reply:
Doesn't look like it, I'll remove it. It was generated by the migration schematic.